src/Microsoft.PowerShell.Commands.Utility/commands/utility/Csv.cs

```diff
@@ -20,92 +22,69 @@ internal CSVHelper(char delimiter)
         /// </summary>
         internal char Delimiter { get; } = ',';
 
+        internal char Quote { get; } = '"';
```

Comment:
I don't see a value for this. Please use const `'"'` in code - it is more readable.